concepticondata/conceptlists/Huttenlocher-1979-225.tsv

```diff
@@ -0,0 +1,226 @@
+Huttenlocher-1979-225-1	1	1	coat	noun	clothing	1.A	146	603	COAT
+Huttenlocher-1979-225-2	2	2	dog	noun	animals	1.A	146	2009	DOG
+Huttenlocher-1979-225-3	3	3	eye	noun	body parts	1.A	146	1248	EYE
+Huttenlocher-1979-225-4	4	4	chair	noun	grooming	1.A	146	1476	CHAIR
+Huttenlocher-1979-225-5	5	5	sock	noun	clothing	1.A	146	1522	SOCK
+Huttenlocher-1979-225-6	6	6	horse	noun	animals	1.A	146	615	HORSE
+Huttenlocher-1979-225-7	7	7	hair	noun	body parts	1.A	146	2648	HAIR (HEAD)
+Huttenlocher-1979-225-8	8	8	wall	noun	grooming	1.A	146	933	WALL (OF HOUSE)
+Huttenlocher-1979-225-9	9	9	hat	noun	clothing	1.A	146	771	HAT
+Huttenlocher-1979-225-10	10	10	cat	noun	animals	1.A	146	1208	CAT
+Huttenlocher-1979-225-11	11	11	mouth	noun	body parts	1.A	146	674	MOUTH
+Huttenlocher-1979-225-12	12	12	door	noun	grooming	1.A	146	1567	DOOR
+Huttenlocher-1979-225-13	13	13	shoe	noun	clothing	1.A	146	1381	SHOE
+Huttenlocher-1979-225-14	14	14	sheep	noun	animals	1.A	146	1331	SHEEP
+Huttenlocher-1979-225-15	15	15	hand	noun	body parts	1.A	146	1277	HAND
+Huttenlocher-1979-225-16	16	16	bed	noun	grooming	1.A	146	1663	BED
+Huttenlocher-1979-225-17	17	17	pants	noun	clothing	1.A	146	809	TROUSERS
+Huttenlocher-1979-225-18	18	18	cow	noun	animals	1.A	146	1007	COW
+Huttenlocher-1979-225-19	19	19	nose	noun	body parts	1.A	146	1221	NOSE
+Huttenlocher-1979-225-20	20	20	desk	noun	grooming	1.A	146	1775	TABLE
+Huttenlocher-1979-225-21	21	21	shirt	noun	clothing	1.A	146	1622	SHIRT
+Huttenlocher-1979-225-22	22	22	duck	noun	animals	1.A	146	1360	DUCK
+Huttenlocher-1979-225-23	23	23	ear	noun	body parts	1.A	146	1247	EAR
+Huttenlocher-1979-225-24	24	24	house	noun	grooming	1.A	146	1252	HOUSE
+Huttenlocher-1979-225-25	25	1	walk	verb	verbs of movement	1.B	146	1443	WALK
+Huttenlocher-1979-225-26	26	2	remain	verb	stative verbs	1.B	146	987	REMAIN
+Huttenlocher-1979-225-27	27	3	say	verb	verbs of saying	1.B	146	1458	SAY
+Huttenlocher-1979-225-28	28	4	skip	verb	verbs of movement	1.B	146		
+Huttenlocher-1979-225-29	29	5	sleep	verb	stative verbs	1.B	146	1585	SLEEP
+Huttenlocher-1979-225-30	30	6	report	verb	verbs of saying	1.B	146		
+Huttenlocher-1979-225-31	31	7	trot	verb	verbs of movement	1.B	146		
+Huttenlocher-1979-225-32	32	8	wait	verb	stative verbs	1.B	146	1590	WAIT (FOR)
+Huttenlocher-1979-225-33	33	9	tell	verb	verbs of saying	1.B	146	1711	TELL
+Huttenlocher-1979-225-34	34	10	jump	verb	verbs of movement	1.B	146	1829	JUMP
+Huttenlocher-1979-225-35	35	11	sit	verb	stative verbs	1.B	146	1416	SIT
+Huttenlocher-1979-225-36	36	12	question	verb	verbs of saying	1.B	146	1309	ASK (INQUIRE)
+Huttenlocher-1979-225-37	37	13	run	verb	verbs of movement	1.B	146	1519	RUN
+Huttenlocher-1979-225-38	38	14	relax	verb	stative verbs	1.B	146	168	REST
+Huttenlocher-1979-225-39	39	15	call	verb	verbs of saying	1.B	146	1084	CALL
+Huttenlocher-1979-225-40	40	16	hike	verb	verbs of movement	1.B	146		
+Huttenlocher-1979-225-41	41	17	stay	verb	stative verbs	1.B	146	987	REMAIN
```

Comment:
I would unmap it (and its duplicate Huttenlocher-1979-225-62) because of 'remain' in Huttenlocher-1979-225-26 above.

Reply:
duplicates are allowed, as we have three experiemnts.

Reply:
sorry, yes you are right